.bazelrc

```diff
@@ -101,6 +101,9 @@ build:remote --bes_results_url="https://source.cloud.google.com/results/invocati
 # Set remote caching settings
 build:remote --remote_accept_cached=true
 
+# When building remotely, runfile trees are created from in-memory manifests
+build: remote --build_runfile_manifests=false
```

Comment:
is the extra space here harmless? (the ones above all say `build:remote`)

Reply:
Whoops